### PR TITLE
Add arm64 image build support

### PR DESCRIPTION
**Pull request by jianghang8421**
**Included**  
Add support for building image for ARM64.

**Related issue**  
https://github.com/rancher/rancher/issues/16461  

**Build Environment**  
The  ARM64 image needs to be built in an ARM64 environment.

### CODE REVIEW

File: scripts/package [hunk 1]
```diff
@@ -7,6 +7,8 @@ ARCH=${ARCH:-"amd64"}
 SUFFIX=""
 [ "${ARCH}" != "amd64" ] && SUFFIX="_${ARCH}"
 
+
```
Comment (niusmallnan) [inline scripts/package]: ？？

**Comment by niusmallnan** [comment]
Could provide the docker image size after this change?
We hope it won't increase too much.

**Comment by jianghang8421** [comment]
@niusmallnan  
Before this change, the amd64 docker image size is 124MB. 
After this change, the amd64 image size is also 124MB and the arm64 image size is 117MB.

File: scripts/package (hunk 1 above)
Comment (jianghang8421) [reply scripts/package]: Modified

**Comment by loganhz** [comment]
link to https://github.com/rancher/rancher/issues/16518

**Comment by jianghang8421** [comment]
Close this PR and I will submit new PR using drone1.0 for CICD

**Pull request closed without merging**